src/Pages/Auth/Login/index.js

```diff
@@ -1,6 +1,9 @@
-import React from "react"
+import React, { useState } from "react"
 import { useHistory } from "react-router-dom"
 import PropTypes from "prop-types"
+import { form, input } from "./styles"
+import { cx } from "emotion"
+import Input from "../../../components/Input"
```

Comment:
You can use absolute imports in order to avoid many ../../../. The absolute imports start from the src folder
```suggestion
import Input from "components/Input"
```